.pre-commit-config.yaml

```diff
@@ -9,3 +9,26 @@ repos:
     hooks:
     - id: flake8
       args: ['--config=setup.cfg',  '--exclude=./cla_backend/settings/*,./docs/*']
+-   repo: https://github.com/jazzband/pip-tools
```

Comment:
what happens if you edit requirements-base.in? You mentioned it in the training.

Reply:
@jogilder I did further work after the video to make sure it recompiles after base.in is changed, see this commit https://github.com/ministryofjustice/cla_backend/pull/838/commits/7981e31716420644b6188891251aee3dfdc68d7f